e2e/tests/plugins/notebook/tags.e2e.spec.js

```diff
@@ -0,0 +1,123 @@
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+/*
+This test suite is dedicated to tests which verify form functionality.
+*/
+
+const { test, expect } = require('@playwright/test');
+
+test.describe('Tagging in Notebooks', () => {
+    async function createNotebookAndEntry(page) {
+        //Go to baseURL
+        await page.goto('/', { waitUntil: 'networkidle' });
+
+        // Click button:has-text("Create")
+        await page.locator('button:has-text("Create")').click();
+        // Click li:has-text("Notebook")
+        await page.locator('li:has-text("Notebook")').click();
+        // Click button:has-text("OK")
+        await Promise.all([
+            page.waitForNavigation(/*{ url: 'http://localhost:8080/#/browse/mine/b8a33dd9-34c1-4981-82c7-0b5f45301fde?tc.mode=fixed&tc.startBound=1655456181296&tc.endBound=1655457981296&tc.timeSystem=utc&view=notebook-vue&pageId=ede28c83-206a-45cb-bcb0-cf5ba889af51&sectionId=4f75da12-0fa1-45aa-9174-1f21cb710fc0' }*/),
+            page.locator('button:has-text("OK")').click()
+        ]);
+
+        // Click text=To start a new entry, click here or drag and drop any object
+        await page.locator('text=To start a new entry, click here or drag and drop any object').click();
+    }
+
+    async function createNotebookEntryAndTags(page) {
+        await createNotebookAndEntry(page);
+        // Click text=To start a new entry, click here or drag and drop any object
+        await page.locator('button:has-text("Add Tag")').click();
+
+        // Click [placeholder="Type to select tag"]
+        await page.locator('[placeholder="Type to select tag"]').click();
+        // Click text=Driving
+        await page.locator('text=Driving').click();
+
+        // Click button:has-text("Add Tag")
+        await page.locator('button:has-text("Add Tag")').click();
+        // Click [placeholder="Type to select tag"]
+        await page.locator('[placeholder="Type to select tag"]').click();
+        // Click text=Science
+        await page.locator('text=Science').click();
+    }
+
+    test('Can load tags', async ({ page }) => {
+        await createNotebookAndEntry(page);
+        // Click text=To start a new entry, click here or drag and drop any object
+        await page.locator('button:has-text("Add Tag")').click();
+
+        // Click [placeholder="Type to select tag"]
+        await page.locator('[placeholder="Type to select tag"]').click();
+
+        await expect(page.locator('.c-input--autocomplete__options')).toContainText("Science");
```

Comment:
@scottbell We try to avoid building dependencies on styling classes from our tests, and we did this by defining classes prefixed with `js-` which by design would have no styling associated with it. 


On the one hand, it's unavoidable that our tests have dependencies on our markup, so changes to the view will eventually break tests, however we handle this, and that's ok. But I think building dependencies on styling classes makes our tests particularly brittle.

@unlikelyzero @charlesh88 Is using `js-` prefixes still the right approach? We have also discussed using arias for for a similar purpose.

Reply:
@akhenry we should avoid using class names if it all possible.